src/main/java/de/tum/in/www1/artemis/repository/ExerciseRepository.java

```diff
@@ -139,25 +139,25 @@ AND EXISTS (
             WHERE
             e.course.studentGroupName member of u.groups
             AND e.course.teachingAssistantGroupName not member of u.groups
-            AND e.course.editorGroupName not member of u.groups
+            AND (e.course.editorGroupName IS NULL OR  e.course.editorGroupName not member of u.groups)
```

Comment:
Style nit: Two spaces after `OR`.
```suggestion
            AND (e.course.editorGroupName IS NULL OR e.course.editorGroupName not member of u.groups)
```